### PR TITLE
Adding themes

### PR DESCRIPTION
**Pull request by gewlar**
Introducing a simple theme loader, that is able to style the element by using css variables.

Simply pass the theme parameter to the polyring webcomponent. You can even add your own themes and then pass their url to the component

### CODE REVIEW

File: src/assets/scripts/embed.njk [hunk 1]
```diff
@@ -3,7 +3,14 @@ permalink: 'embed.js'
 ---
 const WEBRING_DATA_URL = "https://xyquadrat.ch/polyring/data/members.json"
```
Comment (XYQuadrat) [inline src/assets/scripts/embed.njk:4]: If we migrate to `{{meta.dataUrl}}` (which sounds like a good idea), we can drop this, since it's not used anymore.

File: src/assets/scripts/embed.njk [hunk 2]
```diff
@@ -87,6 +98,21 @@ class WebringBanner extends HTMLElement {
...
+            font-variant-ligatures:normal;
+            font-variant-caps:normal;
+            font-variant-numeric:normal;
+            font-variant-east-asian:normal;
+            text-align:start
+            font-weight="400";
+            font-size="16.933";
+            font-family="Comfortaa";
+            letter-spacing="0" word-spacing="0";
+
```
Comment (XYQuadrat) [inline src/assets/scripts/embed.njk:113]: Unnecessary whitespace.

File: src/assets/scripts/embed.njk [hunk 3]
```diff
@@ -146,7 +172,12 @@ class WebringBanner extends HTMLElement {
           ${this.getStyles()}
           <div class="webring-banner">
             <div class="webring-banner__header">
-              <img class="webring-banner__image" src="${this.image}" alt="" />
+              <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 64 64" class="webring-banner__image">
```
Comment (XYQuadrat) [inline src/assets/scripts/embed.njk:175]: I'm not 100% happy with this solution since it means one has to adjust this path separately from the .svg if changes are made, but I guess it's not that bad if we get it to look nice with all theme combos.

A possible different solution would be to add an additional parameter which would go into `filter: invert(<param>)`.

File: src/assets/scripts/embed.njk [hunk 4]
```diff
@@ -87,6 +98,21 @@ class WebringBanner extends HTMLElement {
             border-radius: 50%;
           }
 
+          .webring-banner_text {
+            line-height:0%;
+            -inkscape-font-specification:'Comfortaa, Normal';
+            font-variant-ligatures:normal;
+            font-variant-caps:normal;
+            font-variant-numeric:normal;
+            font-variant-east-asian:normal;
```
Comment (XYQuadrat) [inline src/assets/scripts/embed.njk:107]: Are those needed? I'm not up-to-date with web stuff, but they seem potentially superfluous.

**Review by XYQuadrat — COMMENTED** [review]
Awesome! :100: This truly lifts the widget to the next level.

File: src/assets/scripts/embed.njk (hunk 1 above)
Comment (gewlar) [reply src/assets/scripts/embed.njk:4]: Ah had that removed in another edit. I'll push that change, too

File: src/assets/scripts/embed.njk (hunk 4 above)
Comment (gewlar) [reply src/assets/scripts/embed.njk:107]: Most likely not, I just copied them over from the svg.

File: src/assets/scripts/embed.njk (hunk 3 above)
Comment (gewlar) [reply src/assets/scripts/embed.njk:175]: I don't like the filter option, as it's restricting possibilities quite a lot. Having to embed the svg to make it work is a bit annoying. I'm sure we can do something during the build though, which takes the svg file and embeds it.

Comment (XYQuadrat) [reply src/assets/scripts/embed.njk:168]: Makes sense. That can be tackled in another commit though; for now this is great so let's roll with it.
